docs/src/api.md

```diff
@@ -1,24 +1,5 @@
-# ChainRules API Documentation
+# API Documentation
 
-```@docs
-ChainRules.frule
-ChainRules.rrule
-ChainRules.AbstractRule
-ChainRules.Rule
-ChainRules.DNERule
-ChainRules.WirtingerRule
-ChainRules.accumulate
-ChainRules.accumulate!
-ChainRules.store!
-```
-
-```@docs
-ChainRules.AbstractDifferential
-ChainRules.extern
-ChainRules.Casted
-ChainRules.Wirtinger
-ChainRules.Thunk
-ChainRules.Zero
-ChainRules.DNE
-ChainRules.One
+```@autodocs
+Modules = [ChainRulesCore, ChainRules]
```

Comment:
I think this might wind up being a bit general. Just built the docs and the API section now includes some very cholesky-specific stuff that it really probably shouldn't. Is there a way to exclude unwanted docstrings when doing this?

Reply:
we can set `private = false` to ignore non-exported stuff... but for some reason when i tried this locally the `chol_*` and `level*` stuff still seems to get included

For now ChainRules doesn't define any user-facing behaviour AFAIU, so I might just exclude it altogether and only just do `Modules = [ChainRulesCore]`